vmdb/tools/fix_auth/models.rb

```diff
@@ -93,6 +94,23 @@ def self.contenders(_ = nil)
     end
   end
 
+  # newer databases do not have this column
+  class FixUser < ActiveRecord::Base
+    include FixAuth::AuthModel
+    self.password_columns = %w(password)
+    self.table_name = "users"
+  end
+
+  # migration will fix, no need to do this here
+  # # newer databases do not have passwords here
+  # class FixZone < ActiveRecord::Base
```

Comment:
would like to document this case.
Not necessary since a migration will upgrade this column and upgrade to using the v2_key in the process.

Since `zones.settings` is formatted differently from other yaml columns, it blows up.